packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -68,6 +178,17 @@ function ChangePassword({
     }
   };
 
+  /** Return null if Auth.getCurrentAuthenticatedUser is still in progress  */
```

Comment:
Think this can just be an inline comment, same with other one line comments

Reply:
Done through other commits.